performance_regression_test.py

```diff
@@ -99,7 +105,35 @@ def get_test_xml(self, result):
 
         return test_content
 
-    def display_results(self, results):
+    def generate_stats_json(self, results, cmds=[]):
+        metrics = {}
+
+        for p in self.params.iteritems():
+            metrics[p[1]] = p[2]
+
+        # we use cmds
+        del metrics['stress_cmd']
+        for cmd in cmds:
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+
+        metrics['test_name'] = self.params.id.name
+        metrics['stats'] = results
```

Comment:
The results contains result of `multiple loader` * `multiple c-s client` in each loader. We might need to aggregate the data before reporting to ES server, or aggregate them in analysis in ES server.

@roydahan any suggestion?